src/main/java/org/jenkinsci/plugins/googleplayandroidpublisher/ApkPublisher.java

```diff
@@ -60,6 +60,7 @@
     private String trackName;
     private String rolloutPercentage;
     private RecentChanges[] recentChangeList;
+    private String inAppUpdatePriority;
```

Comment:
Here and elsewhere, we should try to represent it as an integer, either nullable or initialized to -1 to indicate no value (and then omit it in the edit)

Edit: actually I see that you allow it to be expanded by variables 🤔 I guess that means it does need to be a string then